.github/workflows/ci.yaml

```diff
@@ -2,6 +2,12 @@ name: ci
 on:
   pull_request:
   push:
+    branches:
+      - main
+      - release/*
+      - snapshot/*
+    tags:
+      - v*
```

Comment:
Are there any other special case branches or tags we'd want to run on "push"?

Reply:
I don't see other cases for the moment.

But I think it will be interesting to do the same for other ci files. So we have a consistent way to run the different job regardless their purpose.

Reply:
Good call! The other files were set up to only run on push to `main`. I'll adjust them to include these other items too.

Reply:
Done!